haystack/components/preprocessors/csv_document_cleaner.py

```diff
@@ -0,0 +1,119 @@
+# SPDX-FileCopyrightText: 2022-present deepset GmbH <info@deepset.ai>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+from io import StringIO
+from typing import Dict, List
+
+import pandas as pd
```

Comment:
since we are probably removing pandas as a hard dependency, we should start importing it in a lazy way